pubtools/pulplib/_impl/client/client.py

```diff
@@ -203,6 +203,36 @@ def search_repository(self, criteria=None):
             Repository, "repositories", criteria=criteria, search_options=search_options
         )
 
+    def search_repository_content(self, repository_id, type_ids, criteria=None):
```

Comment:
so, you mainly exposed two public APIs, one is Client.search_repository_content and another is Repository.search_content and they basically did the same thing, am I right?
Do you think it's kinda redundant? Like publish, we only exposed it in Repository, not Client, since this operation is about repository directly.

Reply:
Right, you can either use Client.search_repository_content() to look up a repo's units or you can call search_content() from a repository object. I don't see the harm in leaving Client's method public -- it may help in some, probably infrequent, use cases. We can hide it, though, if there's reason to.